src/css/custom.css

```
@@ -221,13 +221,26 @@ svg.landingLogo path {
     display: block;
 }
 
+.img-small {
+    width: 40%;
```

IMO, we shouldn't scale up small images. Instead, we should set the max width of large images.

```suggestion
    max-width: 40%;
```